### DISCUSSION TITLE
Stuck on developer account verification

### DISCUSSION THREAD

**Discussion by extendmygamelist** [microsoft/MicrosoftEdge-Extensions#148] [Q&A]
Good day! It has already been over 2 weeks and I still see the "Your developer account has not been verified yet. To publish your extension, please complete account verification on the Account Settings page." warning. I have filled in all needed info, already submitted 7 versions of the extension, but still can't publish it. My account is individual, the process should take around 5 business days according to the documentation... Could you please point me to what to do next to resolve the issue?

**Comment by yiyungent** [comment]
me too

```
Cannot publish extension. Your developer account verification is either pending or rejected. Please complete your account verification on the Account Settings page before you begin publishing.
```

![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/16939388/91838c97-0993-4694-a334-b5f49f7802d2)

**Comment by Unlicensed-ZZZ** [comment]
Hello!
The same problem. Registered individual account on 25.06,2024. No feedback till now. I wonder if I didn't complete any form, there is any mistake in data, Microsoft don't need extension developers any more?
The process takes so long? Answer something!

**Comment by liambolling** [comment]
Nothing works. Customer support said my email was generic (should not contain contact@) but I used my name to signup so.. something is very wrong here.

**Comment by ManikanthMSFT** [comment]
Hi Dev's, thank you for reaching out to us. To ensure we can assist you effectively, please contact us at **ext_dev_support@microsoft.com** along with your Store ID's. Our team is ready to provide the support you need.

**Comment by liambolling** [comment]
It's been a week and I received an email asking me to reverify (which is broken on the website in Edge and Chrome). I've contacted support. This is still not working / nobody can verify their employment (why is this even needed?) Google doesn't make me do this to create a Chrome extension.

**Comment by Unlicensed-ZZZ** [comment]
I also wrote to ext_dev_support@microsoft.com, but nothing happened. It's a kind of black hole?

**Reply by ManikanthMSFT** [reply]
@Unlicensed-ZZZ, we apologize for any inconvenience. If you haven’t received an update, could you please resubmit your email? This will help us assist you more efficiently.

**Reply by Unlicensed-ZZZ** [reply]
Hello! OK, did it. Wrote e-mail in "Legal info" once again and saved it. Must I do anything else with it?
By the way, I have written to suggested mail (ext_dev_support@microsoft.com) on 29 of July, 21:58. Exactly from from the mail address, that I gave in "Legal info". May be this would help you to "...assist you more efficiently..."? I have no respond ether from registration service, nor from your "ext_dev_support@".
Why do we have this quest with Microsoft?! Registering with Google took me about a day for successful result.

**Reply by Unlicensed-ZZZ** [reply]
May be giving it here (unlicensed.zzz@gmail.com) would speed up this process?

**Reply by ManikanthMSFT** [reply]
Hey @Unlicensed-ZZZ, we tried to verify your email, but we haven't received any email from it. So, we request you to email us if you are still facing an issue with account verification.

**Reply by Unlicensed-ZZZ** [reply]
Hello! I already did it - wrote to suggested e-mail "ext_dev_support@microsoft.com"- First - on 29 July 21:58, second - 22 August 22:17/ No reply on them.
Ok, I shall give another e-mail in registration form - "unlicensed@mail.ru". Let's see - would it change anything...

**Reply by Unlicensed-ZZZ** [reply]
Hello! I sent the letter 30.08.2024 and have got reply from you group (Manikanth Yellapu) 02.09.2024. No changes in account activation, no further replies till now. What is going wrong?

**Reply by Unlicensed-ZZZ** [reply]
06.02.2025. No changes in account activation. No changes in Support request 2410040040003472. Half a year past...

**Comment by liambolling** [comment]
Why is this closed? It's an ongoing issue and no one at Microsoft has solved it? @ManikanthMSFT??

**Comment by LimbersMay** [comment]
I have the same issue, I can't publish my extension

**Comment by ManikanthMSFT** [comment]
Hey @LimbersMay, could you please contact us at ext_dev_support@microsoft.com along with your Store ID's. Our team is ready to provide the support you need.

**Comment by liambolling** [comment]
Why is this closed if it's clearly an ongoing issue with multiple people? 5 upvotes, 5 separate comments, etc and no explanation of the problem or resolution.

**Comment by kshmir** [comment]
+1

**Reply by ManikanthMSFT** [reply]
Hey @kshmir, could you please provide more details about your concern along with the seller ID? This will help us verify the issue and assist you more effectively.

**Comment by Thorvarium** [comment]
I am SHOCKED about this. Why is it so hard to publish extensions on Edge? I guess you guys don't want people publishing extensions. Go look at Chrome or Firefox process so yall can learn something.

**Reply by ManikanthMSFT** [reply]
Hey @Thorvarium, could you please provide more details about your concern along with the seller ID? This will help us verify the issue and assist you more effectively.

Thank you!

**Comment by hnzzstw** [comment]
+1

**Reply by ManikanthMSFT** [reply]
Hey @hnzzstw , could you please provide more details about your concern along with the seller ID? This will help us verify the issue and assist you more effectively.

**Reply by hnzzstw** [reply]
@ManikanthMSFT   Seller ID:89790520   Please help me check the reason why the account cannot be verified.

**Reply by hnzzstw** [reply]
@ManikanthMSFT  please check the Seller ID:89790520.

**Reply by ManikanthMSFT** [reply]
@hnzzstw, we request you to create an SR ticket using the [Partner Center](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpartner.microsoft.com%2Fen-us%2Fsupport%3Fstage%3D1&data=05%7C02%7Cv-ymanikanth%40microsoft.com%7C029090a799854726977408dcde19e4d8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638629450038419937%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=qDbCZVyO0zIqaN%2FID9vp788iJi%2BgB1aBKLl5jQyBd4I%3D&reserved=0). This will route you to the appropriate team best suited to address your concerns.

If you still face any issue, please contact us for further assistance.

**Comment by tofrankie** [comment]
+1

**Comment by eximido** [comment]
Registered one month ago to the Microsoft Edge Add-ons program in order to port some extensions that I created for Chrome. And, like others here, I stuck at "Your developer account has not been verified yet. To publish your extension, please complete account verification on the Account Settings page.", while there's literally NOTHING at Account Settings page to proceed with. No warnings, no errors, no messages - no nothing! And I have no idea what verification it even talks about. I'm an individual developer, what can be verified here? For so long?

Then I wrote a letter to [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com) regarding the issue. It was around two weeks ago already. And guess what? No reply ever since.

So, um... Nice work, guys. Seems MS Edge just doesn't want more extensions in their store as it ignores the developers, like, completely.

**Reply by eximido** [reply]
My Seller ID is 90357100. Not sure anyone would actually do anything with that though.

**Reply by eximido** [reply]
FYI.

I've decided to do an experiment and created a new blank Microsoft account (my real account is 10+ years old and has multiple products registered to it). So I registered a brand new @outlook.com email address just for that. Then I registered this blank account to the Microsoft Edge Add-ons program using all COMPLETELY FAKE personal data including names, addresses and contact info, and also I selected the United States as a country instead of my real country there.

And, you know, my assumption turned out to be absolutely correct:
A completely blank account with all completely fake data got approved IMMEDIATELY.

Meanwhile my real account with all real data is stuck in this "validation" for weeks and months.

Thank you, Microsoft, for this.

**Reply by ManikanthMSFT** [reply]
Hi @eximido, could you please confirm the country from which you registered the account on your original account?
